README.md

```diff
@@ -1,512 +1,50 @@
 
-```java
-// Publishing message of type String
-channel.publish("greeting", "Hello World!", new CompletionListener() {
-	@Override
-	public void onSuccess() {
-		System.out.println("Message successfully sent");
-	}
-
-	@Override
-	public void onError(ErrorInfo reason) {
-		System.err.println("Unable to publish message; err = " + reason.message);
-	}
-});
-
-// Publishing message of type JsonElement
-JsonObject jsonElement = new JsonObject();
-
-Map<String, String> inputMap = new HashMap<String, String>();
-inputMap.put("name", "Joe");
-inputMap.put("surename", "Doe");
-
-for (Map.Entry<String, String> entry : inputMap.entrySet()) {
-    jsonElement.addProperty(entry.getKey(), entry.getValue());
-}
-
-channel.publish("greeting", message, new CompletionListener() {
-	@Override
-	public void onSuccess() {
-		System.out.println("Message successfully sent");
-	}
-
-	@Override
-	public void onError(ErrorInfo reason) {
-		System.err.println("Unable to publish message; err = " + reason.message);
-	}
-});
-```
-
-#### Querying the history
-
-```java
-PaginatedResult<Message> result = channel.history(null);
-
-System.out.println(result.items().length + " messages received in first page");
-while(result.hasNext()) {
-	result = result.getNext();
-	System.out.println(result.items().length + " messages received in next page");
-}
-```
-
-#### Presence on a channel
-
-```java
-channel.presence.enter("john.doe", new CompletionListener() {
-	@Override
-	public void onSuccess() {
-		// Successfully entered to the channel
-	}
-
-	@Override
-	public void onError(ErrorInfo reason) {
-		// Failed to enter channel
-	}
-});
-```
-
-#### Querying the presence history
-
-```java
-PaginatedResult<PresenceMessage> result = channel.presence.history(null);
-
-System.out.println(result.items().length + " messages received in first page");
-while(result.hasNext()) {
-	result = result.getNext();
-	System.out.println(result.items().length + " messages received in next page");
-}
-```
-
-#### Channel state
-
-`Channel` extends `EventEmitter` that emits channel state changes, and listening those events is possible with `ChannelStateListener`
-
-```java
-ChannelStateListener listener = new ChannelStateListener() {
-	@Override
-  public void onChannelStateChanged(ChannelStateChange stateChange) {
-    System.out.println("Channel state changed to " + stateChange.current.name());
-    if (stateChange.reason != null)
-        System.out.println("Channel state error" + stateChange.reason.message);
-  }
-};
-```
-
-You can register using
-
-```java
-channel.on(listener);
-```
+The following platforms are supported:
 
-and after you are done listening channel state events, you can unregister using
-```java
-channel.off(listener);
-```
+| Platform | Support |
+|----------|---------|
+| Java | >=8 (JRE 1.8 or later) |
+| Kotlin | >= 2.1.10 |
```

Comment:
Let's make it `>= 1.8`, our SDK is pure Java, so we support all Kotlin version since 0.0.1 :) but Kotlin version < 1.8 are more than 2 y.o. so I think we should advise to use newer versions